### PR TITLE
add character amount display

### PR DESCRIPTION
**Pull request by Satttoshi**
pls check

- characters left display in form

### CODE REVIEW

**Comment by Satttoshi** [comment]
Dont mention Styling pls

File: js/index.js [hunk 1]
```diff
@@ -84,3 +84,36 @@ function showAnswer(event, data) {
...
+const inputA = document.querySelector(`[data-js="inputa"]`);
+
+const amountLeftQ = document.querySelector(`[data-js="amount-left-q"]`);
+const amountLeftA = document.querySelector(`[data-js="amount-left-a"]`);
+
+const maxLengthQ = inputQ.getAttribute("maxlength");
+const maxLengthA = inputA.getAttribute("maxlength");
+
+// For Q
+const updateAmountLeftQ = (value) => {
```
Comment (toshydev) [inline js/index.js:98]: You could save some lines with a counter function but if it works, it works💪

**Review by toshydev — APPROVED** [review]
One suggestion to save lines.
Rest LGTM!

**Review by toshydev — CHANGES_REQUESTED** [review]
Navigation container is not responsive. 
Maybe change width to 100 vw?

**Pull request merged**